app/components/arara/text_field_component.rb

```diff
@@ -20,12 +20,20 @@ def initialize(variant: "default", label: nil, helper_text: nil, max_length: nil
       @has_error = has_error
       @user_variant = "outlined" if textarea
       @type = type
+      @placeholder = placeholder
     end
 
     def has_error?
       @has_error
     end
 
+    def label_class
+      string_value = value.try(:to_s)
+      return "mdc-floating-label" if string_value.nil?
+      return "mdc-floating-label mdc-floating-label--float-above" unless string_value.empty?
+      "mdc-floating-label"
+    end
+
```

Comment:
is this needed? I couldn't find in the `erb` template

Reply:
I'll remove and make a research on where this is being used, maybe is somewhere on the system here!

Reply:
@fernandes Removed!